mne/_freesurfer.py

```diff
@@ -537,3 +539,62 @@ def _get_lut(fname=None):
     lut = {d[0]: np.array(lut[d[0]], dtype=d[1]) for d in dtype}
     assert len(lut['name']) > 0
     return lut
+
+
+def get_head_surface(surf, subject, subjects_dir, bem=None):
+    """Get a head surface from the Freesurfer subject directory.
+
+    Parameters
+    ----------
+    surf : str
+        The name of the surface 'auto', 'head', 'outer_skin', 'head-dense'
+        or 'seghead'.
+    %(subject)s
+    %(subjects_dir)s
+
+    Returns
+    -------
+    verts : ndarray
+        The surface vertices or None if ``surf='auto'`` and none is found.
+    triangles : ndarray
+        The vertex triangles or None if ``surf='auto'`` and none is found.
+
+    Notes
+    -----
+    .. versionadded: 0.24
+    """
+    _check_option(
+        'surf', surf, ('auto', 'head', 'outer_skin', 'head-dense', 'seghead'))
+    if surf in ('auto', 'head', 'outer_skin'):
+        if bem is not None:
+            try:
+                return _bem_find_surface(bem, 'head')
+            except RuntimeError:
+                logger.info('Could not find the surface for '
+                            'head in the provided BEM model, '
+                            'looking in the subject directory.')
+    if subject is None:
+        if surf == 'auto':
+            return
```

Comment:
This does not match the return type in the docstring (to do that, you'd want to return `(None, None)`. It seems like for public function this should raise an error.

This is in a private module, so maybe you want this to stay private? I think it should be private... In that case it's better to (still) have it prefixed by `_` in that case (even though the module also is) because if we ever import it in some public module (like `mne.bem` or something) it will become "accidentally public". So underscores on function names even in `_`-prefixed modules are still useful/helpful. It also makes it immediately clear in the places that it's used that we can change the API if we want.